agent/src/main/java/com/testsigma/agent/controllers/HomeController.java

```diff
@@ -84,4 +88,8 @@ public ResponseEntity<String> register(@PathVariable("uuid") String uuid,
     }
     return response;
   }
+  private ArrayList<AgentBrowser> getUniqueBrowsersList(ArrayList<AgentBrowser> browsers){
+    Set<AgentBrowser> set = browsers.stream().collect(Collectors.toSet());
```

Comment:
@nixonakash01 If there are 3 versions of the same browser, which version are you picking ? Ensure that you pick the latest one

Reply:
@tarun-testsigma only same version of the browser is displayed as per the bug, so it will remove the replicated ones. refer jira screenshot for the browser versions displayed.

Reply:
Lets say if edge browser has 3 versions: 111, 111 and 110, which one will it remove ?

Reply:
It is saved in set so, if the versions are 111,111 and 110, then 111 and 110 will be displayed once, and not 111,111,110. does not add value to Set if value already exists @tarun-testsigma

Reply:
No. There should only be only one browser version. The edge browser with the latest version only should be displayed

Reply:
actually, there will not more then one version for a single browser. and this is what is implemented in cloud also.